.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,161 @@
+                if "node_modules" in root:
+                    continue
+                for file_name in files:
+                    if (
+                        file_name.endswith(".ts")
+                        and not file_name.endswith(".test.ts")
+                        and not file_name.endswith(".spec.ts")
+                    ):
+                        file_path = os.path.join(root, file_name)
+                        if has_code_coverage_disable(file_path):
+                            print(
+                                f"""File {file_path} contains code coverage disable statement."""
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)